src/stylesheets/vendors/bootstrap.scss

```diff
@@ -0,0 +1,13 @@
+@import '~bootstrap/scss/functions';
```

Comment:
@noor-ahmed Should we add all in the bootstrap here and comment to those components which are we don't use it?
Once we want to use that we just un-comment that line. 🤔